students/grzegorz_bajorski/lesson_02_flow_control/the_second_maximum.py

```diff
@@ -0,0 +1,14 @@
+maximum = 0
+max_second = 0
+x = 1
+
+while x != 0:
+    x = int(input())
+    if x > maximum:
+        max_second = maximum
+        maximum = x
+    elif x < maximum and x > max_second:
+        max_second = x
+
+print(max_second)
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.